README.md

```
@@ -1,4 +1,4 @@
-
+https://media.giphy.com/media/l1J9EdzfOSgfyueLm/giphy.gif
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)